src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml

```diff
@@ -13,9 +13,20 @@
                 <ResourceDictionary Source="../../SharedUI/SharedResources.xaml" />
             </ResourceDictionary.MergedDictionaries>
 
+            <ContextMenu x:Key="RowMenu" DataContext="{Binding PlacementTarget.DataContext, RelativeSource={RelativeSource Self}}">
+                <MenuItem Header="Open in browser" 
```

Comment:
`Open in SonarQube`/`Open in SonarCloud` would be a better description; it's not immediately obvious what `open in browser` is going to do.

Reply:
Do you mean the literal string `Open in SonarQube / SonarCloud`, or a dynamic string based on the server type?

Reply:
I meant a dynamic string.

Reply:
Update: I suggest committing it as it as, and I'll create a fit-and-finish improvement ticket for making the label dynamic.

Reply:
I was just about to write a comment regarding this. It would have to be a NotifyPropertyChanged text, that would listen to binding changes. I agree about the suggestion of separating this to another ticket.